src/collections/_documentation/apm/apm-glossary.md

```diff
@@ -0,0 +1,170 @@
+
+An example of a span that describes a call to an external service:
+
+    {
+          "trace_id": "a0fa8803753e40fd8124b21eeb2986b5",
+          "parent_span_id": "9c2a6db8c79068a2",
+          "span_id": "8c931f4740435fb8",
+          "start_timestamp": 1563890702.134,
+          "same_process_as_parent": true,
+          "description": "http://httpbin.org/base64/aGVsbG8gd29ybGQK GET",
+          "tags": { "http.status_code": 200, "error": false },
+          "timestamp": "1563890702.934",
+          "op": "http",
+          "data": {
+            "url": "http://httpbin.org/base64/aGVsbG8gd29ybGQK",
+            "status_code": 200,
+            "reason": "OK",
+            "method": "GET"
+          }
+        }
+
+## Trace
+
+A trace is a collection of spans that have the same `trace_id` attribute. Traces consist of all spans generated by the distributed system during the processing of the input data. Traces are not explicitly started or finished. The start and end of a trace are defined by the transactions and spans participating in the trace.
+
+### Propagation
+
+Processing of the input data in the distributed environment means that an arbitrary number of the systems/microservices can be involved.
+
+To connect all the spans that were created during the processing of the input data, we need to be able to propagate the basic identifier. By forwarding `trace_id` and `parent_span_id` as data travels through various applications in a microservice ecosystem, we can reconstruct data flows later.
+
+Trace metadata is propagated through http headers that are included in the http request to the external system.
+
+## Transaction
+
+There already was a concept of transaction in the Sentry SDKs before we started with APM.
+
+Transaction is an event attribute that provides initial context to the error report. This makes it possible for the error to be linked to a specific part of the system. An example of the transaction name can be “/users/” which is the name for the endpoint that received the request that failed during the processing.
+
+With APM, the transaction is a parent span to all spans created during the processing of the request in the single server.
+
+Once the request is processed, the transaction is sent together with all child spans to Sentry.
+
+### Event Types
+
+The SDKs already collect a lot of information about the processing of the request. In the case of an error, the SDK attaches the info to the error report to provide rich context of what happened before the error appeared -- information like tags and breadcrumbs.
+
+If the error doesn’t appear during the processing of the request, we discard the context data once processing is complete.
+
+We want to leverage this context collection to build spans, and once a request is finished, we want to send the data to Sentry.
+
+Example of the transaction event type:
+
+    {     
```

Comment:
Can we trim the examples down? A lot of what is contained in this one is irrelevant for APM. I would remove the following keys:

* `modules`
* `extra`
* `server_name`
* `platform`
* `breadcrumbs` (those are not supposed to be sent!)
* `sdk`